src/table/table.component.ts

```diff
@@ -313,6 +313,7 @@ import { I18n } from "./../i18n/i18n.module";
 				<tr *ngIf="!model.isRowFiltered(i)"
 					(click)="onRowSelect(i)"
 					[attr.data-parent-row]="(model.isRowExpandable(i) ? 'true' : null)"
+					class="{{model.rowsClass[i] ? model.rowsClass[i] : null}}"
```

Comment:
Is it possible to bind directly to `class`? so - 
```suggestion
					[class]="model.rowsClass[i] ? model.rowsClass[i] : null"
```

or 

```suggestion
					[attr.class]="model.rowsClass[i] ? model.rowsClass[i] : null"
```

Reply:
Sure, makes sense!